mmengine/runner/runner.py

```diff
@@ -1705,6 +1705,19 @@ def train(self) -> nn.Module:
 
         # initialize the model weights
         self._init_model_weights()
+
+        # try to enable fast_conv_bn_eval feature
+        fast_conv_bn_eval = self.cfg.get('fast_conv_bn_eval', None)
+        if fast_conv_bn_eval is not None:
+            from mmengine.model.fast_conv_bn_eval import \
+                turn_on_fast_conv_bn_eval
```

Comment:
This import statement can be moved to the head of this file.

Reply:
This import statement might raise RuntimeError. It is safer to keep it here to minimize the influence of code not turning on `fast_conv_bn_eval` feature.